components/molecule/select/src/index.scss

```diff
@@ -95,6 +95,11 @@ $c-molecule-select-disabled: inherit !default;
       &-container {
         @extend %sui-atom-input-select-focus;
         border: $bdw-s solid $c-primary;
+
+        .sui-AtomInput-input,
+        .sui-AtomInput--withTags {
+          border-color: transparent;
```

Comment:
border-width: 0 to remove additional borders?

Reply:
If you change the width of the border, it can cause a side effect (little movement) of the layout, isn't it?

Reply:
true